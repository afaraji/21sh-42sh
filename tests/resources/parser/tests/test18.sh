#!/bin/sh

#echo "[****** Testing for ;ls ******]"
../Minishell -t parser ";ls"

#echo "[****** Testing for ;;ls ******]"
../Minishell -t parser ";;ls"

#echo "[****** Testing for ls; ******]"
../Minishell -t parser "ls;;"

#echo "[****** Testing for ;ls ******]"
../Minishell -t parser "ls;"

#echo "[****** Testing for ;ls ******]"
../Minishell -t parser ";"

#echo "[****** Testing for ;ls ******]"
../Minishell -t parser ";;"

#echo "[****** Testing for ;ls ******]"
../Minishell -t parser "qsdsdqs;;qsdqsdq"
