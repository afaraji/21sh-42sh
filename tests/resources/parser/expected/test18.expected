Parser: Syntax error near ';'
Parser: Syntax error near ';;'
Parser: Syntax error near ';;'
Parser say OK
Parser: Syntax error near ';'
Parser: Syntax error near ';;'
Parser: Syntax error near ';;'
